.yamllint.yml

```diff
@@ -0,0 +1,7 @@
+---
+extends: default
+
+rules:
+    line-length:
+        max: 99
+        level: warning
```

Comment:
I would suggest to ignore `document-starts` for now:
`document-start: disable`

Reply:
> Would it make more sense to add this as a ctest to run over the compiled profiles/groups/rules?
Yes, and it would catch errors from our Jinja templating. 

> I would suggest to ignore document-starts for now:
Good idea.

Reply:
> Would it make more sense to add this as a ctest to run over the compiled profiles/groups/rules?

After trying it out (using `find build/*/groups build/*/rules build/*/profiles -name "*.yml" | xargs yamllint -c .yamllint.yml`) I don't think there is much value on using on the compiled rules. As it seems those are create with `yaml.dump` Ideally we would validate right after jinja runs.